qobuz_importer.user.js

```diff
@@ -133,30 +135,27 @@ function insertLink(release) {
   mbUI.slideDown();
 }
 
+// Hook all XMLHttpRequest to use the data fetched by the official web-player.
+(function() {
+    const send = XMLHttpRequest.prototype.send
+    XMLHttpRequest.prototype.send = function() {
+        this.addEventListener('load', function() {
+            var wsUrl = 'https://www.qobuz.com/api.json/0.2/album/get?album_id=';
```

Comment:
Why is `app_id` removed ?

Reply:
We do not need it, as the new code piggybacks on web player. There were two exactly same requests for the album data with different app_ids. I thought that hooking to the first request saves one of those, and wont need a new app_id.

I think the current app_id has been deactivated, as the request was denied (that was reported in the issue).